static/css/style.css

```diff
@@ -346,12 +346,26 @@ button:hover i.fab.fa-slack.fa-.p-orange,
     background: white;
 }
 
+.final_score_table thead tr th {
+    top: -1px;
+}
+
 .final_score_table thead tr th:last-child, 
 .final_score_table tbody tr td:last-child {
     position: sticky;
     right: -1px;
     z-index: 2;
     background: white;
+    font-weight: 600;
+}
+
+.table_titles th {
+    position: sticky!important;
+    top: -1px!important;
+    left: 200px;
+    background-color: white;
```

Comment:
We should try to stick with using the CSS variables where possible.
`var(--white)` instead of just `white`.
This should ideally also be updated for lines: 315, 320, 346, 496, and 507.

The same is true for any instance of `lightgray`, since we have 2 root variables for `var(--s-grey)` and/or `var(--bg-grey)`.
These lines should ideally be fixed: 368, 525, 534